### DISCUSSION TITLE
Ip route default

### DISCUSSION THREAD

**Discussion by BartOo** [fabianishere/udm-iptv#101] [Q&A]
When trying to install this my internet gets broken.
Running ip route gives the following: 

default via 10.28.96.1 dev iptv metric 233
10.28.96.0/20 dev iptv proto kernel scope link src 10.28.109.247
45.92.225.0/24 dev eth8.141 proto kernel scope link src 45.92.225.195
192.168.0.0/24 dev br0 proto kernel scope link src 192.168.0.1
192.168.3.0/24 dev br148 proto kernel scope link src 192.168.3.1
192.168.55.0/24 dev br55 proto kernel scope link src 192.168.55.1

deleting the default restores internet. IPTV works but begin gemist, pause tv not.
Is this normal?

root@ubnt:/# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN Ranges: 213.75.0.0/16 217.166.0.0/16 10.28.96.0/20 10.28.110.105/20 217.166.0.0/16 224.0.0.0/8 185.24.175.0/24 185.41.48.0/24 217.166.226.0/24
WAN VLAN: 148 (dev iptv)
WAN DHCP options: -O staticroutes -V IPTV_RG
LAN Interfaces: br148
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
33: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.28.109.247/20 scope global iptv
       valid_lft forever preferred_lft forever
default via 10.28.96.1 metric 233
10.28.96.0/20 proto kernel scope link src 10.28.109.247
=== Service Logs ===
Mar 18 20:27:35 ubnt systemd[1]: Started IPTV support for the UniFi Dream Machine.
Mar 18 20:27:35 ubnt udm-iptv[31596]: Device iptv already exists.. Deleting device
Mar 18 20:27:35 ubnt udm-iptv[31596]: Obtaining IP address for VLAN interface
Mar 18 20:27:35 ubnt udm-iptv[31596]: udhcpc (v1.22.1) started
Mar 18 20:27:35 ubnt udm-iptv[31596]: Sending discover...
Mar 18 20:27:37 ubnt udm-iptv[31596]: Sending select for 10.28.109.247...
Mar 18 20:27:37 ubnt udm-iptv[31596]: Lease of 10.28.109.247 obtained, lease time 345600
Mar 18 20:27:37 ubnt udm-iptv[31596]: NATing IPTV network ranges (if necessary)
Mar 18 20:27:37 ubnt udm-iptv[31596]: Setting up igmpproxy
Mar 18 20:27:37 ubnt udm-iptv[31596]: Starting igmpproxy

**Comment by fabianishere** [comment]
What's your ISP?

Given that you have specified `-O staticroutes`, it seems that your ISP's DHCP server is not responding with the right DHCP options.

**Comment by BartOo** [comment]
Thanks for the response, it is fiber.nl (glasvezel)
It gives me an ip for iptv but makers it default, the other on 141 is the internet ip

**Reply by fabianishere** [reply]
To temporarily fix this issue, you can comment out the following lines in `/usr/lib/udm-iptv/udhcp.hook`:
https://github.com/fabianishere/udm-iptv/blob/dfe265552d0124a3b11aa620838f9b0f91ea08c8/udhcpc.hook#L85-L88

**Comment by BartOo** [comment]
thanks changing values doesn't result in not working internet anymore.

Now trying to get begin gemist working ;)

**Reply by bvk007** [reply]
Good luck with that...

To give you some pointers, you need to build RTSP conntrack modules for your UDM (do some reading in the tweakers topic here : https://gathering.tweakers.net/forum/list_message/71024430#71024430) 

And since your providers doesn't provide your with static routes (I have the same issues) you need to create some static routes over the IPTV gateway to get it going.

I don't have it working either. but @reinierjh does...but he's on Tweak and they do provide option 121 (classless static route) in their DHCP response.
